packages/api-reference/src/features/toolbar/ApiReferenceToolbarShare.vue

```diff
@@ -0,0 +1,107 @@
+<script lang="ts" setup>
+import {
+  ScalarFormSection,
+  ScalarIconButton,
+  ScalarTextInput,
+} from '@scalar/components'
+import {
+  ScalarIconBracketsCurly,
+  ScalarIconCopy,
+  ScalarIconFileMd,
+  ScalarIconGitBranch,
+  ScalarIconGlobeSimple,
+  ScalarIconLockSimple,
+  ScalarIconWarningOctagon,
+} from '@scalar/icons'
+import { type ScalarIconComponent } from '@scalar/icons/types'
+import type { ApiReferenceConfiguration } from '@scalar/types'
+import { useClipboard } from '@scalar/use-hooks/useClipboard'
+import { computed } from 'vue'
+
+import ApiReferenceToolbarBlurb from '@/features/toolbar/ApiReferenceToolbarBlurb.vue'
+import ApiReferenceToolbarPopover from '@/features/toolbar/ApiReferenceToolbarPopover.vue'
+
+const FEATURES = [
+  { icon: ScalarIconLockSimple, label: 'Password Protected' },
+  { icon: ScalarIconGlobeSimple, label: 'Custom Domains' },
+  { icon: ScalarIconWarningOctagon, label: 'Spectral Rules' },
+  { icon: ScalarIconGitBranch, label: 'Bi-directional Git' },
+  { icon: ScalarIconFileMd, label: 'Markdown Files' },
+  { icon: ScalarIconBracketsCurly, label: 'Json Schema Support' },
+] as const satisfies ReadonlyArray<{
+  icon: ScalarIconComponent
+  label: string
+}>
+
+const { configuration } = defineProps<{
+  configuration?: Partial<ApiReferenceConfiguration>
+}>()
+
+const editorShareHref = computed<string>(() => {
+  const encodedConfig = encodeURIComponent(JSON.stringify(configuration))
+  return `https://editor.scalar.com/preview?config=${encodedConfig}`
```

Comment:
I think we should make a `@/consts/url.ts` and put all the URLs there

* it’s so easy to not match all with the URLs when doing search & replace
* sometimes it’s great to just check URLs all at once, and not stumble about a single outdated URL

Reply:
Done!